### PR TITLE
Release v1.1.0

### PR DESCRIPTION
**Pull request by asmacdo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+
+- In `ansible-operator` and `helm-operator` run commands, print git commit when logging version information. ([#3849](https://github.com/operator-framework/operator-sdk/pull/3849))
+- For Ansible/Helm, fix multigroup message. ([#3822](https://github.com/operator-framework/operator-sdk/pull/3822))
+- For Helm-based Operators, cleanup the file `.gitignore` in order to not have invalid instructions for the type. ([#3810](https://github.com/operator-framework/operator-sdk/pull/3810))
+- Updated scorecard-test-kuttl image to kuttl v0.6.1. ([#3659](https://github.com/operator-framework/operator-sdk/pull/3659))
+- Updated scorecard-test-kuttl image to kuttl v0.5.2. ([#3659](https://github.com/operator-framework/operator-sdk/pull/3659))
+- Updated scorecard-test-kuttl image to use latest kuttl. ([#3711](https://github.com/operator-framework/operator-sdk/pull/3711))
+- Updated the Makefile to build multiarch versions of the scorecard-test-kuttl and custom-scorecard-tests images to match the scorecard-test image. ([#3821](https://github.com/operator-framework/operator-sdk/pull/3821))
+
+### Bug Fixes
```
Comment (camilamacedo86) [inline CHANGELOG.md:24]: Only Missing  https://github.com/operator-framework/operator-sdk/pull/3983




12:38

**Comment by asmacdo** [comment]
@camilamacedo86 great catch. /hold

File: CHANGELOG.md (hunk 1 above)
Comment (asmacdo) [reply CHANGELOG.md:24]: 3983 is now included.

**Review by camilamacedo86 — APPROVED** [review]
/lgtm 

Great 👍

**Review by jmrodri — APPROVED** [review]
/lgtm

**Pull request merged**